### DISCUSSION TITLE
Change icon for file with pending save

### DISCUSSION THREAD

**Discussion by scascalespAV** [microsoft/vscode-discussions#244] [Extension Development QnA] [answered]
Hello
In notepad++ when you editing a file the * symbol appears near the tab tittle.  It's marked as 'pending to save'

Do you know some extension in VSC to replicate it? I see 'M' right to tittle but always appear (modified or not)

Thanks

**Comment by alefragnani** [comment, marked as answer]
Hi @scascalespAV ,

VS Code already does that natively. The `x` button (Close) in tab title is changed to a dot. 

Hope this helps

**Comment by scascalespAV** [comment]
I can see the change! perfect! Thanks!
